docs/application/tizen-studio/native-tools/ui-builder/custom-ui-component.md

```diff
@@ -45,7 +45,7 @@ A custom component requires an EDJ resource file (`[ComponentName].edj`) and an
     |-----------|-----------|-------------|----------------------------------------|-------|--------|------------------------------------|
     | -           | <component> | name          | Component  identifier, shown on the Palette, Outline view, and Properties view. This attribute must be identical to the XML file  name. | String  | Yes      | -                                    |
     | -           | <component> | description   | Description  shown when you mouse over the custom UI component on the Palette | String  | No       | null                                 |
-    | -           | <component> | group         | group block defined in the EDC file for the component.      For more information see [Group Block](../../../org.tizen.guides/html/native/ui/efl/learn_edc_group_n.htm). | String  | Yes      | -                                    |
+    | -           | <component> | group         | group block defined in the EDC file for the component.      For more information see [Group Block]( ../../native/guides/ui/efl/learn-edc-intro.md ). | String  | Yes      | -                                    |
```

Comment:
It looks `../../../native/guides/ui/efl/learn-edc-group.md` is correct.

Reply:
done